app/src/main/res/values-de/strings.xml

```diff
@@ -549,11 +549,20 @@ Signatur: %5$s</string>
     <string name="discard_changes_question">Änderungen verwerfen?</string>
     <string name="discard">Verwerfen</string>
     <string name="keep_editing">Weiter bearbeiten</string>
+
+    <string name="channel_description_general">Nächste Vorlesung, News, Erinnerung über Studienbeiträge</string>
+    <string name="channel_description_cafeteria">Aktuelles Tagesmenü</string>
```

Comment:
in der Mensa

Reply:
As above, enough context via the channel name.